src/__init__.py_base

```diff
@@ -0,0 +1,11 @@
+import os
+
+import elife_api
+
+__version__ = '0.0.1'
```

Comment:
We haven't versioned this repo, though I think we could.

Reply:
Happy to change the format or version, but it does need a value to pin to in https://github.com/elifesciences/api-raml/blob/b05910d25fa34df276d10d81f3c3d12bef09739f/setup.py#L7 . Is '0.0.1' OK for now?